content/graph/MaximumClique.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * Author: chilli, Haobin Ni
+ * Date: 2019-05-10
+ * License: CC0
+ * Source: https://en.wikipedia.org/wiki/MaxCliqueDyn_maximum_clique_algorithm,
+ * https://github.com/FTRobbin/Dreadnought-Standard-Code-Library/blob/11c861e55a73be9c32a799bfe4398e0c62c2da15/improve/MaximumClique.cpp
+ * Description: Finds the maximum clique of a graph. Can be used to find the
+ * maximum independent set by finding the clique of the complement graph.
+ * Time: Runs in about ~1s for n=155 and worst case random graphs (p=.95). Runs
+ * faster for sparse graphs.
+ * Status: fuzz-tested
+ */
+struct Maxclique {
+	double limit=0.025, pk=0;
+	struct Vertex { int i, d; Vertex(int i) : i(i), d(0) {} };
```

Comment:
declare this as `struct Vertex { int i, d = 0; }` and construct instances using `{i}`